pkg/sdk/testint/dynamic_table_integration_test.go

```diff
@@ -54,7 +54,9 @@ func TestInt_DynamicTableCreateAndDrop(t *testing.T) {
 		}
 		query := "select id from " + tableTest.ID().FullyQualifiedName()
 		comment := random.Comment()
-		err := client.DynamicTables.Create(ctx, sdk.NewCreateDynamicTableRequest(name, testWarehouse(t).ID(), targetLag, query).WithOrReplace(true).WithComment(&comment))
+		refreshMode := "FULL"
```

Comment:
It would be nice to add a separate test for the default creation (without explicit refreshMode and initialize).

Then, for both these tests we should add checks for the show command (because of the extraction from text we do on the resource level).